jujugui/static/gui/src/app/components/unit-details/_unit-details.scss

```diff
@@ -0,0 +1,14 @@
+.unit-details {
+    &__properties {
+        padding: 10px 20px;
+
+        p {
+            @extend %stagger-fade-in;
+            margin: 0 0 10px 0;
```

Comment:
Could this be converted to `margin-bottom: 10px` to stop overriding wanted styles later